EIPS/eip-1706.md

```diff
@@ -3,12 +3,14 @@ eip: 1706
 title: Disable SSTORE with gasleft lower than call stipend
 author: Alex Forshtat <alex@tabookey.com>, Yoav Weiss <yoav@tabookey.com>
 discussions-to: https://github.com/alex-forshtat-tbk/EIPs/issues/1
-status: Draft
+status: Superseded
```

Comment:
According to EIP-1 "Superseded" applies to "Final" EIPs, e.g. the fitting one here would be "Abandoned" with perhaps a note that EIP-2200 is the preferred way.

Reply:
Changed status to 'abandoned'